notebooks/example.ipynb

```diff
@@ -200,6 +199,228 @@
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[2024-07-18T19:27:40]     INFO - fixtures/config_test_file_a.nml\n",
+      "[2024-07-18T19:27:40]     INFO + fixtures/config_test_file_b.nml\n",
+      "[2024-07-18T19:27:40]     INFO ---------------------------------------------------------------------\n",
+      "[2024-07-18T19:27:40]     INFO foo:               n:  - 88 + 99\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "False"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "file_a = \"./fixtures/config_test_file_a.nml\"\n",
+    "\n",
+    "different_file_b = \"./fixtures/config_test_file_b.nml\"\n",
+    "\n",
+    "config.compare(file_a, different_file_b)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c16c38d0",
+   "metadata": {},
+   "source": [
+    "The `config()` method returns `False` to denote the files are different. The UW logger shows the difference, of one file containing `n = 88`, and one file containing `n = 99`.\n",
+    "\n",
+    "Now to compare two semantically equivalent files:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "370149a3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "&foo n=88, s=\"string\" /"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "cat ./fixtures/config_test_file_a.nml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "bd444fba",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "&FOO\n",
+      "  S = \"string\"\n",
+      "  N = 88\n",
+      "/"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "cat ./fixtures/config_test_file_c.nml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "9d658b72",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[2024-07-18T19:27:40]     INFO - fixtures/config_test_file_a.nml\n",
+      "[2024-07-18T19:27:40]     INFO + fixtures/config_test_file_c.nml\n",
+      "[2024-07-18T19:27:40]     INFO ---------------------------------------------------------------------\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "file_a = \"./fixtures/config_test_file_a.nml\"\n",
+    "\n",
+    "identical_file_c = \"./fixtures/config_test_file_c.nml\"\n",
+    "\n",
+    "config.compare(file_a, identical_file_c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b84db243",
+   "metadata": {},
+   "source": [
+    "The `config()` method returns `True` to denote the files are semantically equivalent."
```

Comment:
Here `config` is the object. Maybe this should read "The `compare()` method...".

The same applies for the `False` case above.

Reply:
Corrected